src/index.css

```diff
@@ -2,3 +2,371 @@
   min-height: 100vh;
   height: 100vh;
 }
+
+
+head,
+body {
+  background-color: #ecf0f5;
+}
+.content-wrapper {
+  min-height: initial !important;
+}
+.bg-auth {
```

Comment:
I'm also not sure if we need all of the CSS here since I _believe_ it's part of AdminLTE which we're already loading in though I could be wrong.